requirements.in

```diff
@@ -86,3 +86,13 @@ raven[flask]
 # Load testing.
 locustio
 pyzmq
+
+#Impact retour emploi
+matplotlib
+openpyxl
+pygal
+pygal_maps_fr
+matplotlib
+imgkit
+xlsxwriter
+cairosvg
```

Comment:
c'est chaud d'importer autant de libs (comme matplotlib) en production alors qu'elles sont utiles seulement pour l'IRE

on paye le prix de la non séparation de l'importer vs le frontend/api

j'ai pas de solution mais je veux bien l'avis de @celine-m-s 

à noter: Michel pour son projet ROME NAF avait séparé ses requirements dans son dossier spécifique ROME_NAF à la racine du repo, c'était pas mal

Reply:
C'est vrai que ce n'est pas très efficace...
Peut-être qu'on peut créer un requirements.in spécifique à l'importer et installer ces dépendances dans chaque commande make qui les nécessite ?
Ou bien, si c'est Ansible qui lance les commandes make et installe l'environnement, on peut les installer à ce moment-là ?